### PR TITLE
Create Readme

### PR DESCRIPTION
**Pull request by tonycarpenter21**
What (if any) features are you implementing?
- Add a README

What (if anything) did you refactor?
- Nothing

Were there any issues that arose?
- No

Is there anything that you need from your teammate?
- No

Any other comments, questions, or concerns?
- No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,109 @@
...
-### `npm start`
+*Tired of pretending to like ugly art in museums? We are too.*
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Art From The Heart is a web application that allows users to overcome internal and external biases to answer the only question that matters: Do I like this work of art? 
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+The homepage greets a user with categories of art pulled directly from the Art Institute of Chicago. Upon clicking a category, a random work of art will be displayed for a user to rate up or down. Only after they rate the work of art will the details about the author, creation date, description of the work, etc. be shown to the user. This allows people to view and enjoy art without the implications of certain biases being forced upon them such as ___ ___ or ____. After they have rated the art they can then choose to see another art piece from that category or they can choose to go home and select another category.
```
Comment (mr-ryan12) [inline README.md:22]: Excellent description of our app's purpose!

File: README.md [hunk 2]
```diff
@@ -1,46 +1,109 @@
...
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+```bash
+1. Clone down this Repository using `git clone`
+2. Next, `cd` into the cloned repository and `npm install` to install library dependencies
+3. Finally, run `npm start` and open up localhost at the port indicated to view the webpage (http://localhost:3000/)
+```
+Alternatively, you can access the deployed site [HERE](https://art-from-the-heart-mr-ryan12.vercel.app/)
+    
+## Features
```
Comment (mr-ryan12) [inline README.md:51]: Maybe we could add the full terminal command "git clone git@github.com:mr-ryan12/art-from-the-heart.git" and an instruction to close the local server "CTRL + c (mac)"??? Glad the npm install works!

File: README.md [hunk 3]
```diff
@@ -1,46 +1,109 @@
...
+
+A user can select an individual movie to view more details. They can access the movie's trailer, the runtime, year of release, description, rating, and genre(s) that specific movie falls into.
+
+Art From the Heart is set up to be tablet and mobile responsive and resizes nicely for different size screens:
+
+*IMAGE GOES HERE*
+
+Art From the Heart also features a Dark Mode and can easily be toggled by the icon in the navigation bar:
+
+*IMAGE GOES HERE*
```
Comment (mr-ryan12) [inline README.md:84]: This looks great! We'll be sure to add those gifs!

File: README.md [hunk 4]
```diff
@@ -1,46 +1,109 @@
-# Getting Started with Create React App
+<img width="100" alt="Art From The Heart" src="https://user-images.githubusercontent.com/88450229/156020128-b1e39b76-0121-42b3-8b55-2c450d7e781a.png"> Art From The Heart <img width="100" alt="Art From The Heart" src="https://user-images.githubusercontent.com/88450229/156020128-b1e39b76-0121-42b3-8b55-2c450d7e781a.png">
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+- Click [HERE](https://frontend.turing.edu/projects/module-3/stretch.html) to view the project spec
+- Click [HERE](https://art-from-the-heart-mr-ryan12.vercel.app/) to view the deployed site
 
-## Available Scripts
+## Table of Contents
```
Comment (StevieAmb) [inline README.md:6]: Awesome job with the structure of this!

File: README.md [hunk 5]
```diff
@@ -1,46 +1,109 @@
...
+- [See the Site in Action](#see-the-site-in-action)
+- [Future Goals](#future-goals)
+- [Created By](#created-by)
+- [Credit](#credit)
 
-In the project directory, you can run:
+## Abstract
 
-### `npm start`
+*Tired of pretending to like ugly art in museums? We are too.*
```
Comment (StevieAmb) [inline README.md:18]: Haha!

**Review by StevieAmb — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1,46 +1,109 @@
...
+
+Individual Art View:
+
+*IMAGE GOES HERE*
+
+Art Details View:
+
+*IMAGE GOES HERE*
+
+A user can select an individual movie to view more details. They can access the movie's trailer, the runtime, year of release, description, rating, and genre(s) that specific movie falls into.
```
Comment (mr-ryan12) [inline README.md:76]: lol, probably can get this out.

**Pull request merged**